src/taco-dependency-installer/installers/androidSdkInstaller.ts

```diff
@@ -36,11 +37,9 @@ import utilHelper = tacoUtils.UtilHelper;
 class AndroidSdkInstaller extends InstallerBase {
     private static ANDROID_HOME_NAME: string = "ANDROID_HOME";
     private static androidCommand = os.platform() === "win32" ? "android.bat" : "android";
-    private static ANDROID_PACKAGES: string[] = [
-        "tools",
+    private static ANDROID_PACKAGES: string[] = [    // IDs of Android Packages to install. To get the list of available packages, run "android list sdk -u -a -e"
+        // "tools",  // Android SDK comes by default with the tools package, so there is no need to update it. In the future, if we feel we want dependency installer to always install the latest Tools package, then uncomment this.
         "platform-tools",
-        "extra-android-support",
-        "extra-android-m2repository",
         "build-tools-19.1.0",
         "build-tools-21.1.2",
         "build-tools-22.0.1",
```

Comment:
Is there any way to fetch these programmatically and not having to do a release of taco every time a new Android version comes along?